app/views/allies/index.html.erb

```diff
@@ -31,7 +31,19 @@
     </div>
   </div>
 <% end %>
-
+<% if @invited_allies.any? %>
+  <div class="title"><%= t('allies.index.invited') %></div>
+  <div class="spacer"></div>
```

Comment:
We don't have the spacer class anymore. I would use marginTop in the same div where gridThree is.